accounts/templates/accounts/participant_list.html

```diff
@@ -44,16 +52,16 @@
    <div class="row mt-xl visible-md visible-lg visible-xl">
        <div class="col-md-3 col-md-offset-1">
            <strong> Username </strong>
-           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
-           <a aria-label="Sort descending by username" href="{% url 'exp:participant-list' %}?{% query_transform request sort='-given_name' %}"><i aria-hidden="true" class="fa fa-chevron-down" role="button" name="name"></i></a>
+           <a class="pl-sm" aria-label="Sort ascending by username" href="{% url 'exp:participant-list' %}?{% query_transform request page='1' sort='given_name' %}"><i aria-hidden="true" class="fa fa-chevron-up" role="button" name="name"></i></a>
```

Comment:
Then paginating won't work right.  If "sort" is specified as "name", but I am actively paginating my results, going to page 2, page 3, I need it to stay at page one.

This query transform tag sets the url before you even click it.  It's not reacting to a button you just clicked, it just builds the url based on what the current parameters of the page are.